docs/pages/includes/soc2.mdx

```diff
@@ -6,7 +6,7 @@ The audit report covers:
 - Teleport Enterprise, self-hosted
 - Teleport Enterprise, cloud-hosted (SaaS)
 
-The SOC 2 report is available for download at
-[trust.goteleport.com](https://trust.goteleport.com).
+Paying customers and prospects under NDA may download the SOC 2 report at
+[trust.goteleport.com](https://trust.goteleport.com/d/teleport-access-platform-soc-2-type-ii-hipaa/LP5ypb).
```

Comment:
Our next SOC 2 report will be split (separate SOC 2 and HIPAA reports), so not sure if this is the right approach.

Reply:
This permalink will remain pointing to the most current version of the document even if the title changes. `LP5ypb` is the routing info. If we're planning on deprecating this document instead of renaming it when we split -- you're right, this will break.

See further discussion here: https://gravitational.slack.com/archives/C03CW6Z2B51/p1706127028622439

I made the change to a deeplink to address "customers get lost when pointed to trust.goteleport.com" -- hoping a direct link to the document would be more helpful than a link to the general page.